xclim/indices/_threshold.py

```diff
@@ -90,6 +91,43 @@ def cold_spell_days(
     return group.map(rl.windowed_run_count, window=window, dim="time")
 
 
+@declare_units("", tas="[temperature]", thresh="[temperature]")
+def cold_spell_frequency(
+    tas: xarray.DataArray,
+    thresh: str = "-10 degC",
+    window: int = 5,
+    freq: str = "AS-JUL",
+):
+    r"""Cold spell frequency.
+
+    The number of cold spell events, defined as a minimum number of consecutive days with mean daily
```

Comment:
```suggestion
    The number of cold spell events, defined as a sequence of consecutive days with mean daily
```

Reply:
My suggestion is not great, but I thought the original formulation was unclear.

Reply:
better... I will modify cold_spell_days to match as well

Reply:
done...